app/models/media_object.rb

```diff
@@ -387,7 +387,7 @@ def access_control_bulk documents, params
       successes = []
       documents.each do |id|
         media_object = self.find(id)
-        media_object.hidden = params[:hidden] == "true" if params[:hidden].present?
+        media_object.hidden = params[:hidden]
```

Comment:
What if params[:hidden] isn't present? I think it would be more clear if this had the same form as the next line (visibility).